api/HTMLAllCollection.json

```diff
@@ -124,7 +124,7 @@
               "version_added": "32"
             },
             "safari": {
-              "version_added": "10"
+              "version_added": true
```

Comment:
This is changing "10" to true, but I've confirmed with http://software.hixie.ch/utilities/js/live-dom-viewer/?saved=8758 that it was supported in Safari 9, so this data was not correct. Better to leave it as true and come back to it later. (We haven't generally done this, but it's fine to do in rare cases I think.)

Reply:
Or actually, I'll revert this change, since otherwise the commit message doesn't work. This is removing versions, not adding/correcting :)